### PR TITLE
Add dcadec formula

### PR DESCRIPTION
**Pull request by rcombs**

### CODE REVIEW

File: Library/Formula/dcadec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Dcadec < Formula
+  homepage "https://github.com/foo86/dcadec"
+  version "7a5479454156c7a2223c7452f2a965389dce5e67"
+  url "https://github.com/foo86/dcadec/archive/7a5479454156c7a2223c7452f2a965389dce5e67.tar.gz"
```
Comment (mistydemeo) [inline Library/Formula/dcadec.rb]: Sorry, we need a tagged stable release to accept this software. Could you maybe request one from the author?

Comment (DomT4) [inline Library/Formula/dcadec.rb]: Looks like there'll be one available soon: https://github.com/foo86/dcadec/issues/30

File: Library/Formula/dcadec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  version "f5ffe27a25fb55efe68966677863b96223a11186"
+  url "https://github.com/foo86/dcadec/archive/f5ffe27a25fb55efe68966677863b96223a11186.tar.gz"
+  sha256 "970211a0c5bfaf319c6ae08f46faab98cdbc1e21442c4dd2624946fc1ea140eb"
+
+  head "https://github.com/foo86/dcadec.git"
+
+  def install
+    system "make", "all"
+    unless File.exist?("test/samples/README")
+      system "git", "clone", "git@github.com:foo86/dcadec-samples.git", "test/samples"
```
Comment (DomT4) [inline Library/Formula/dcadec.rb]: We need to add this as a resource rather than a `git clone`.

**Comment by DomT4** [comment]
Ping.

**Comment by DomT4** [comment]
This seems to have gone dead. Closing consequently.

If you pick up interest later please feel free to resubmit as a new PR with the comments above addressed, Thanks!

Thank you for the submission!

**Pull request closed without merging**